Packs/Core/Playbooks/playbook-Remote_WMI_Process_Execution.yml

```diff
@@ -0,0 +1,1004 @@
+          "x": 180,
+          "y": 2950
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 033bca88-f784-423c-8795-135a9bfe88a1
+    type: regular
+    task:
+      id: 033bca88-f784-423c-8795-135a9bfe88a1
+      version: -1
+      name: Close the Alert as False Positive
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      closeNotes:
+        simple: The alert has been resolved, confirmed as a False Positive
+      closeReason:
+        simple: Resolved - Handled by the playbook "Remote WMI Process Execution" as False Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1140,
+          "y": 2770
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: c2b18800-ab04-4323-8962-209698d7d91e
+    type: condition
+    task:
+      id: c2b18800-ab04-4323-8962-209698d7d91e
+      version: -1
+      name: Approval Required – Malicious Activity Detected
+      description: |-
+        **Recommendation: Isolation Required – Malicious Activity Detected**
+
+        The detection meets high-confidence criteria for malicious activity based on the following conditions:
+
+        **Matched Verdicts:**
+        * Matches for high-risk profiles: mixed_case_powershell, double_encoding, amsi_techniques, malicious_commands, or powershell_suspicious_network_patterns.
+
+        * Malicious IP address detected.
+        Score >= 30, indicating high confidence probability for malicious behavior.
+
+        * Score >= 10 with a prevention rule detected in the same incident, correlating to malicious activity.
+
+        **Action Required:**
+
+        * Isolate the remote host
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      Isolate:
+      - "18"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2420
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - Isolate
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
```

Comment:
think there is a chance that isolating the remote endpoint might fail.
Consider enhancing task #19, 'Should proceed to isolate the remote endpoint?' by adding checks to verify if the machine is connected and the agent is installed and not isolated. 
Also, Yarden doesn't want us to perform isolation on servers via automatic tasks in  XSIAM.
Please review the isolation conditional task in the 'Credential Dumping using a known tool' playbook.